app/scripts/lib/validate.js

```diff
@@ -21,6 +21,19 @@ define(function (require, exports, module) {
   // URN Regex
   var urnRegEx = /^urn:[a-zA-Z0-9][a-zA-Z0-9-]{1,31}:([a-zA-Z0-9()+,.:=@;$_!*'-]|%[0-9A-Fa-f]{2})+$/;
 
+  // Email regex, accepts punycoded addresses. See:
```

Comment:
Yep, my thinking was that the comment still stands; the regex *does* accept punycoded addresses, and it may be worth pointing that out for future reference.